CONTRIBUTING.md

```diff
@@ -29,11 +29,7 @@ RuboCop GOV.UK is a styleguide, so each rule in each YAML file should have a com
 
 Find out how much effort it will be to adopt your changes in typical GOV.UK repos. This could just be running `rubocop -A` to autocorrect new issues, or it could involve significant manual effort.
 
-You can test changes locally by tweaking the Gemfile or `.gemspec` for the repo e.g.
-
-```
-gem "rubocop-govuk", path: "/govuk/rubocop-govuk"
-```
+You can test changes against an alphagov repo by [running the GitHub worflow](https://github.com/alphagov/rubocop-govuk/actions/workflows/run-against-project.yml) we have configured.
```

Comment:
I've looked in the UI at https://github.com/alphagov/rubocop-govuk/actions/workflows/run-against-project.yml and can't see anywhere to 'run' the workflow, it only shows me past runs. Am I missing something, or is there a permissions issue?

Reply:
I believe (and aren't certain as this is my first one) that it needs to be on the default branch to be there. So there's a hint of trial and error, sorry I should have noted that.